Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/Ibl.azsli

```diff
@@ -38,6 +39,18 @@ real3 GetIblDiffuse(
     return diffuseResponse * albedo * diffuseSample;
 }
 
+real3 EnvBRDFApprox( real3 specularF0, real roughnessLinear, real NdotV )
+{
+    // Environment brdf approximation (Lazarov 2013). Use this approximation if sampling specular env map gets too expensive or is prohibitive on mobile
+    // see https://www.unrealengine.com/en-US/blog/physically-based-shading-on-mobile
+    const real4 c0 = { -1, -0.0275, -0.572, 0.022 };
+    const real4 c1 = { 1, 0.0425, 1.04, -0.04 };
+    real4 r = roughnessLinear * c0 + c1;
+    real a004 = min( r.x * r.x, exp2( -9.28 * NdotV ) ) * r.x + r.y;
+    real2 env = real2( -1.04, 1.04 ) * a004 + r.zw;
+    return (specularF0 * env.x) + (env.y * saturate( 50.0 * specularF0.g ));
```

Comment:
Is this supposed to be taken from the blog version which looks like 
```
half3 EnvBRDFApprox( half3 SpecularColor, half Roughness, half NoV )
{
	const half4 c0 = { -1, -0.0275, -0.572, 0.022 };
	const half4 c1 = { 1, 0.0425, 1.04, -0.04 };
	half4 r = Roughness * c0 + c1;
	half a004 = min( r.x * r.x, exp2( -9.28 * NoV ) ) * r.x + r.y;
	half2 AB = half2( -1.04, 1.04 ) * a004 + r.zw;
	return SpecularColor * AB.x + AB.y;
}
```
?
If so, how did you arrive at the last line?
```
	return SpecularColor * AB.x + AB.y;
	vs
return (specularF0 * env.x) + (env.y * saturate( 50.0 * specularF0.g ));
```

Reply:
saturate( 50.0 * specularF0.g ) is a form of specular occlusion based on the knowledge that no real-world material has a reflectance lower than 2%. So here we are basically occluding the spec a little if you have F0 of less than 2%. I saw this used by Filament and hence I am applying it here. Since this function is not getting used I have left it here but if it is used we should test it to make sure it works properly.